src/seal-parse.cpp

```diff
@@ -61,12 +61,12 @@ void	SealStrDecode	(sealfield *Data)
  **************************************/
 void	SealStrEncode	(sealfield *Data)
 {
-  size_t i,j;
```

Comment:
Downcasting to a smaller size will create infinite loops for large files. Rejected.